src/clusters.tsx

```diff
@@ -45,13 +51,19 @@ export class DaskClusterManager extends Widget {
   /**
    * Create a new Dask cluster manager.
    */
+  status = 'ready';
```

Comment:
Can you implement this with a getter and a private attribute? That would make it so `status` couldn't be set by third parties. It would also be nice to tighten up the typings here, maybe something like `DaskClusterManager.IStatus = 'ready' | 'starting' | 'failed'`?